src/test/java/org/itsallcode/openfasttrace/importer/TestSpecificationListBuilder.java

```diff
@@ -172,4 +176,45 @@ public void testDuplicateIdNotIgnored()
         builder.endSpecificationItem();
         assertThat(builder.getItemCount(), equalTo(2));
     }
+
+    // [utest->dsn~filtering-by-tags-during-import~1]
+    @Test
+    public void testFilterSpecificationItemsByTags()
+    {
+        final SpecificationListBuilder builder = createListBuilderFilteringByTags("client",
+                "server");
+        builder.beginSpecificationItem();
+        final SpecificationItemId idA = SpecificationItemId.createId("dsn", "in-A", 1);
+        builder.setId(idA);
+        builder.addTag("client");
+        builder.addTag("database");
```

Comment:
Maybe change `addTag()` to accept arg `String... tags`. Then this will be shorter.